### ISSUE TITLE
I cannot enable Sickrage on my Synology NAS since DSM 6.0 Update 1

### ISSUE THREAD

**Issue by Blackvibes** [SickRage/SickRage#1497]
I don't know what version, but at least the latest version up to April 15th, friday, 12:45 GMT+1.

-

I updated yesterday to DSM 6.0 Update 1 manually and suddenly half of all my applications were disabled. However, I could reenable all of them, except for Sickrage (Sickbeard-custom; this repo) and Couchpotato. For both I get the message "Failed to run package service" and then immediately stops the app.

I removed Sickbeard-custom and reinstalled it. However, that didn't matter since the application still won't start. As suggested by [this post](https://forum.synology.com/enu/viewtopic.php?t=117245#p429106), install, but enable after a reboot.. isn't working either.

Any suggestions?

-

I did change to HTTPS and updated the ports on my router (80, 443, 5001). Does that matter by any chance? Sick rage did work fine after this change, but after a reboot during the update of DSM 6.0 update 1 everything failed. So I'm unsure it's due to HTTPS/Port change.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
SSH as admin user, run this:
`/var/packages/sickbeard-cusom/scripts/start-stop-status start`

It will fail, just like the package manager.

check what the out put of that and this says:
`/var/packages/sickbeard-cusom/scripts/start-stop-status status`

**Comment by Blackvibes** [comment]
Ok, this will make it weird.

I SSH'd in my machine and used 'sudo su' to run as admin. Then I executed the first command, to start Sickrage, and it succeeded, instead of failing. No comment given. Then I executed the second command (getting the status) which said "SickBeard Custom is running". 

Now I went into DSM and in Package Center it still showed 'Sickrage: stopped'. When I tried to enable it, it again gave the error message "failed to run the package service". However, I could access sickrage strangely on <IP>:8083. 

So.. I can't use Sickrage, but theoretically, on a bit dirty way, can use Sickrage.

So what's this sorcery?
